lang/java/avro/pom.xml

```diff
@@ -164,6 +164,11 @@
       <artifactId>xz</artifactId>
       <version>${tukaani.version}</version>
     </dependency>
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <optional>true</optional>
```

Comment:
If it doesn't get pulled in as a transitive dependency, then that's fine. (But that may need checking.)